core/api/service/chain/chain_jrpc_processor.cpp

```diff
@@ -31,6 +33,12 @@ namespace kagome::api::chain {
     server_->registerHandler("chain_getHeader",
                              Handler<request::GetHeader>(api_));
 
+    server_->registerHandler("chain_subscribeFinalizedHeads",
+                             Handler<request::SubscribeFinalizedHeads>(api_));
+
+    server_->registerHandler("chain_unsubscribeFinalizedHeads",
+                             Handler<request::UnsubscribeFinalizedHeads>(api_));
+
     server_->registerHandler("chain_subscribeNewHeads",
                              Handler<request::SubscribeNewHeads>(api_));
```

Comment:
This is not a mistake from your side, but I just noted that substrate has alias `chain_subscribeNewHead` that does the same. Therefore can we please add in this PR:
```suggestion
    server_->registerHandler("chain_subscribeNewHead",
                             Handler<request::SubscribeNewHeads>(api_));
    server_->registerHandler("chain_unsubscribeNewHead",
                             Handler<request::UnsubscribeNewHeads>(api_));                        
```